applications/nrf_desktop/doc/info.rst

```diff
@@ -0,0 +1,42 @@
+.. _info:
+
+Info module
+###########
+
+Use the Info module to:
+
+* handle the role of the final subscriber for the :ref:`config_channel` events,
```

Comment:
This suggests that the module handles the role, rather than being set as handler of the role. Will this be clear to the final reader?

Reply:
I will leave this information only in implementation details. I think its enough to inform in the introduction that the module is required by the configuration channel.